Makefile

```diff
@@ -161,51 +161,51 @@ endif
 
 
 .PHONY: docker-compose
-docker-compose: ## Run an arbitrary docker-compose command by passing in the Docker Compose profiles in the "profiles" variable, and args in the "args" variable
+docker-compose: ## Run an arbitrary docker compose command by passing in the Docker Compose profiles in the "profiles" variable, and args in the "args" variable
 	# NOTE: The .env file is used to provide environment variable values that replace variables in the compose file
 	#       Because the .env file does not live in the same place as the compose file, we have to tell compose explicitly
 	#       where it is with "--project_directory". Since this is called from the root Makefile, using ./ points to the dir
 	#       of that Makefile
-	docker-compose ${profiles} --project-directory . --file ${docker_compose_file} ${args}
+	docker compose ${profiles} --project-directory . --file ${docker_compose_file} ${args}
```

Comment:
Why did we remove the - between docker compose? I have historically always used docker-compose. Is there a difference? For example: `docker-compose up usaspending-db usaspending-es`

Reply:
@ayubshahab Recent updates to `docker` have added `compose` as a sub-command of `docker` itself instead of being a standalone executable.

Reply:
@ayubshahab here is the doc to help explain: https://docs.docker.com/compose/releases/migrate/